sgf-viewer/src/content.ts

```diff
@@ -1187,11 +1240,23 @@ export const pages: object =
                     "In the first game, the adversary stakes out the center on <a class='clickable' onclick='setMove(`vit-vs-vit-adversary`, 17)'>move 17</a>, for constructing its group inside the cyclic group, which is fully formed by <a class='clickable' onclick='setMove(`vit-vs-vit-adversary`, 49)'>move 49</a>. After that, the victim surrounds that central group, completing the cyclic group on <a class='clickable' onclick='setMove(`vit-vs-vit-adversary`, 248)'>move 248</a>, while the adversary surrounds it from the outside. This leads to a very dense board, with adversary stones positioned low around the edges, and the victim controlling a huge center, mostly filled by the cyclic group and its encirclement. This pattern is typical for this adversary. On <a class='clickable' onclick='setMove(`vit-vs-vit-adversary`, 255)'>move 255</a>, the victim captures the inside group, but after that the adversary reenters the space and establishes a new inside group. Although not universal, this behavior mirrors tactics also observed with the original cyclic adversary. Finally, at <a class='clickable' onclick='setMove(`vit-vs-vit-adversary`, 283)'>move 283</a>, the victim is doomed, and the cyclic group is taken off the board at <a class='clickable' onclick='setMove(`vit-vs-vit-adversary`, 335)'>move 335</a>."
                 ]
+            },
+            {
+                "title": "Human replication of cyclic attack",
+                "dir_name": "vit-vs-human-cyclic",
+                "paths": ["/shared/sgf-viewer-games/vit-vs-human-cyclic.sgf"],
+                "max_games": 1,
+                "adversary": "Kellin Pelrine (author)",
+                "victim": "<code>ViT-victim</code>, 65536 visits, 64 search threads",
+                "description": [
+                    "A Go expert (Kellin Pelrine) was also able to use a cyclic attack to beat our ViT-victim. The game is shown below. Through <a class='clickable' onclick='setMove(`vit-vs-human-cyclic`, 56)'>move 56</a>, White (Kellin) sets up a loosely surrounded square group destined to be the inside of the cyclic group. This follows the shape used in some of the wins of our original cyclic adversary against this victim. In subsequent moves, white gradually surrounds the cyclic group, with a particular focus on making sure the surrounding groups have lots of liberties so that black will have to see its group is in danger early on to save it. Beginning around <a class='clickable' onclick='setMove(`vit-vs-human-cyclic`, 176)'>move 176</a>, white fills in the cyclic group's liberties, as well as pressing black to make the last connections that complete the cyclic shape. Finally, after <a class='clickable' onclick='setMove(`vit-vs-human-cyclic`, 208)'>208</a>, black's fate is sealed. Black plays on several more moves hoping for white to make a mistake, but ultimately resigns."
```

Comment:
```suggestion
                    "A Go expert (Kellin Pelrine) was also able to use a cyclic attack to beat our ViT-victim. The game is shown below. Through <a class='clickable' onclick='setMove(`vit-vs-human-cyclic`, 56)'>move 56</a>, white (Kellin) sets up a loosely surrounded square group destined to be the inside of the cyclic group. This follows the shape used in some of the wins of our original cyclic adversary against this victim. In subsequent moves, white gradually surrounds the cyclic group, with a particular focus on making sure the surrounding groups have lots of liberties so that black will have to see its group is in danger early on to save it. Beginning around <a class='clickable' onclick='setMove(`vit-vs-human-cyclic`, 176)'>move 176</a>, white fills in the cyclic group's liberties, as well as pressing black to make the last connections that complete the cyclic shape. Finally, after <a class='clickable' onclick='setMove(`vit-vs-human-cyclic`, 208)'>208</a>, black's fate is sealed. Black (ViT) plays on several more moves hoping for white to make a mistake, but ultimately resigns."
```